src/Data/Vec/Relation/Binary/Equality/Setoid.agda

```diff
@@ -79,6 +88,12 @@ map-++ : ∀ {b m n} {B : Set b}
 map-++ f []       = ≋-refl
 map-++ f (x ∷ xs) = refl ∷ map-++ f xs
 
+map-[]≔ : ∀ {b n} {B : Set b}
+  i (f : B → A) (xs : Vec B n) p
+  → map f xs [ i ]≔ f p ≋ map f (xs [ i ]≔ p)
```

Comment:
The argument order in the conclusion is `f` then `xs` then `i` then `p`, so I would do the same in the quantifier prefix... and lift the `{b} {B : Set b}` outside past all of them...

Reply:
I changed the order of arguments.